xnat17/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM tomcat:7-jre7-alpine
+MAINTAINER Thomas Phil <thomas@tphil.nl>
+
+RUN apk add --no-cache postgresql-client curl
+
+# Download XNAT
+ENV XNAT_VER 1.7.3.1
+ENV XNAT_HOME /data/xnat/home
+RUN curl -k -L -o xnat-web-${XNAT_VER}.war https://bintray.com/nrgxnat/applications/download_file?file_path=xnat-web-${XNAT_VER}.war
+
+# Setup XNAT
+RUN mkdir -p $CATALINA_HOME/webapps/ROOT && \
```

Comment:
Could we make the name of the webapp a variable, rather than hard-coding it as `ROOT`?